typedb-src/modules/ROOT/pages/admin/configuration.adoc

```diff
@@ -151,23 +161,81 @@ on-disk data in a database, allocate at least *2 GB* of index cache. Allocating
 
 Additionally, we recommend the sum of data and index caches equal to about *20%* of the total memory of the server.
 
-=== Logging configuration
+[#_log]
+=== Log
 
-The log section of the configuration file contains the logging behavior options of TypeDB.
+The `log` section of the configuration file contains the logging options.
 
 There are three subsections:
 
-* `output`: Define destinations to write logs to.
-** User-defined output channel name
-*** `type` -- it's either `file` or `stdout`.
-*** `directory` -- filepath, relative to the server binary. Only available for `type: file`.
-*** `file-size-cap` -- maximum size of a log file. If the log file reaches the limit, a new file in the same directory
-    will be started. This is similar to the `maxsize` config option in logrotate. Only available for `type: file`.
-*** `archives-size-cap` -- maximum size of all log files. If the total size of all log files in the directory reaches
-    the limit, the oldest one gets removed. Only available for `type: file`.
-* `logger`: Set up logging for modules in TypeDB, along with a log level and output targets (referencing outputs by
-  name defined under the outputs section).
-* `debugger`: Set up TypeDB-specific debuggers. Right now, the only defined type is `type: reasoner`.
+* <<_output,`output`>>
+* <<_logger,`logger`>>
+* <<_debugger,`debugger`>>
+
+[#_output]
+==== Output
+
+`output` subsection defines destinations to write logs to.
+
+* User-defined output channel name
+
+** `type` -- it's either `file` or `stdout`.
+** `base-dir` -- filepath, relative to the server binary. Only available for `type: file`.
+** `file-size-limit` -- maximum size of a log file. If the log file reaches the limit, a new file in the same directory
+   will be started. This is similar to the `maxsize` config option in logrotate. Only available for `type: file`.
+** `archive-grouping` -- configures the rollover and naming policy of archives produced by the logger. +
+   Possible value variants are as follows:
+
+*** `minute` or `minutes`
+//yyyyMMdd-HHmm - internal filename format of a log archive
+*** `hour` or `hours`
+*** `day` or `days`
+*** `week` or `weeks`
+//yyyy-ww
+*** `month` or `months`
+*** `year` or `years`
+
+// Filename template:
+// typedb + fileDateFormat(outputType.archiveGrouping()) + `.0.` log.gz
+// For example, `typedb_202306.0.log.gz`.
+
+** `archive-age-limit` -- configures how long archive files are kept. +
+   Old archives are only deleted when new ones are produced. +
+   The value should be a positive integer, followed by one of the following values for units:
+
+*** `minute` or `minutes`
+*** `hour` or `hours`
+*** `day` or `days`
+*** `week` or `weeks`
+*** `month` or `months`
+*** `year` or `years`
+
+** `archives-size-limit` -- maximum size of all log files. If the total size of all log files in the directory reaches
```

Comment:
Why do you outline the valid arguments for the above options, but not for this one?

Reply:
Good point.
Fixed in https://github.com/vaticle/typedb-docs/pull/702/commits/5b1d872601bf4d586af5d4aecacd6fb735df7ee2